torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -204,6 +204,7 @@ def __init__(
         mutated_arg_names: List[str],  # see [Note: clone mutated buffers]
         optimize_mem,
         heuristic_type,
+        reset_to_zero_arg_names: Optional[List[str]] = None,
```

Comment:
It would be better to move this to the end of the param list: to accommodate for the unlikely but possible case of the args being passed positionally somewhere in the client code.